framework/infrastructure/traffic_director.py

```diff
@@ -129,11 +131,14 @@ def __init__(
         self.firewall_rule: Optional[GcpResource] = None
         self.firewall_rule_ipv6: Optional[GcpResource] = None
         self.target_proxy: Optional[GcpResource] = None
+        self.target_proxy_ipv6: Optional[GcpResource] = None
         # TODO(sergiitk): remove this flag once target proxy resource loaded
         self.target_proxy_is_http: bool = False
         self.alternative_target_proxy: Optional[GcpResource] = None
         self.forwarding_rule: Optional[GcpResource] = None
+        self.forwarding_rule_ipv6: Optional[GcpResource] = None
         self.alternative_forwarding_rule: Optional[GcpResource] = None
+        self.enable_dualstack = xds_flags.ENABLE_DUALSTACK.value
```

Comment:
1. We need to move this to `framework.xds_k8s_testcase.XdsKubernetesBaseTestCase.setUpClass`, where the other flags are parsed.
2. Add `enable_dualstack` parameter to the `TrafficDirectorManager` constructor
3. Provide initiate `TrafficDirectorManager` with `enable_dualstack=self.enable_dualstack` in  `framework.xds_k8s_testcase.RegularXdsKubernetesTestCase`

Reply:
Done